src/data_gradients/managers/abstract_manager.py

```diff
@@ -257,3 +259,34 @@ def _create_samples_iterated_warning(self) -> str:
         msg_train = f"Train set: {self._train_iters_done} out of {total_train_samples} samples were analyzed{portion_train}.\n"
         msg_val = f"Validation set: {self._val_iters_done} out of {total_val_samples} samples were analyzed{portion_val}.\n "
         return msg_head + msg_train + msg_val
+
+    @staticmethod
+    def _get_grouped_feature_extractors(default_config_name: str, config_path: str, feature_extractors) -> Dict[str, List[AbstractFeatureExtractor]]:
+        if feature_extractors is None:
+            if config_path is None:
+                config_dir, config_name = None, default_config_name
+            else:
+                config_path = os.path.abspath(config_path)
+                config_dir, config_name = os.path.dirname(config_path), os.path.basename(config_path).split(".")[0]
+            grouped_feature_extractors = load_report_feature_extractors(config_name=config_name, config_dir=config_dir)
```

Comment:
I think I would actually move the if/else logic inside the `load_report_feature_extractors`.